packages/manager/apps/dedicated/client/app/billing/main/billing-main.routes.js

```diff
@@ -1,6 +1,6 @@
 angular
   .module('Billing')
-  .config(($stateProvider) => {
+  .config(($stateProvider, $urlServiceProvider) => {
```

Comment:
```suggestion
  .config(/* @ngInject */($stateProvider, $urlServiceProvider) => {
```

Reply:
Could be handled into a separate Pull Request I think (see results bellow)

```sh
$ git grep "\.config((\$stateProvider" -- "*.js" | wc -l
90
```

Reply:
Maybe we can create a batch PR to fix all of those if that sounds good to you @cbourgois

Reply:
Yep could be done in another PR. 
Just pointing that because you edited the line.